conbench/static/app.css

```diff
@@ -109,3 +109,10 @@ table.dataTable td {
   white-space: nowrap;
   overflow: hidden;
 }
+
+.table-entry {
+  width: -moz-fit-content;
+  width: fit-content;
+  padding: 3px;
+  margin-bottom: 1em;
```

Comment:
How did we decide between using padding vs. margin? 

When I look at the screenshot then I think for just a hand full of rows this look ok/nice, but when displaying many entries (like, 100) then the additional spacing might maybe be wasteful?

Reply:
Hit and trial mostly

- minimum relative margin (Eg: the commit col was touching the cell boundary, seemed little off)